### PR TITLE
chore: add license check

### PR DESCRIPTION
**Pull request by nirgur**
## Related Issues

Fixes <link_to_github_issue>

## Related PRs

| branch       | PR         |
| ------------ | ---------- |
| service a PR | Link to PR |
| service b PR | Link to PR |

## Description

A few sentences describing the overall goals of the pull request's commits.

## Must

- [ ] Tests
- [ ] Documentation (if applicable)

### CODE REVIEW

File: packages/sdks/web-js-sdk/manualLicense.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (dorsha) [inline packages/sdks/web-js-sdk/manualLicense.json:1]: Let's remove this, since it is not MIT

Comment (nirgur) [reply packages/sdks/web-js-sdk/manualLicense.json]: I just checked that it's working
Removed

**Comment by nirgur** [comment]
@dorsha what do we want to do with this PR?

**Comment by dorsha** [comment]
We are still waiting for answers, let's merge this and set the manual license for now.

**Comment by nirgur** [comment]
@dorsha please review

**Pull request merged**